src/library_pthread.js

```diff
@@ -666,7 +666,7 @@ var LibraryPThread = {
     // Deduce which WebGL canvases (HTMLCanvasElements or OffscreenCanvases) should be passed over to the
     // Worker that hosts the spawned pthread.
     // Comma-delimited list of CSS selectors that must identify canvases by IDs: "#canvas1, #canvas2, ..."
-    var transferredCanvasNames = attr ? {{{ makeGetValue('attr', 36, 'i32') }}} : 0;
+    var transferredCanvasNames = attr ? {{{ makeGetValue('attr', 40, `${SIZE_TYPE}`) }}} : 0;
```

Comment:
POINTER_TYPE?   And no need to the backticks here think.  Since `POINTER_TYPE` itself is already a string, right?